### ISSUE TITLE
Does this Project Already Exist?

### ISSUE THREAD

**Issue by wdoug** [codefordenver/encorelink#3]
### Does this project exist?

Have you found solutions for the same challenge that exist already? Are there apps or solutions out there that are already being used to address this issue? If so, how do they fall short of meeting their goal, if at all? Be honest with yourself. Is it worth our time to build something that competes with what’s already out there? Are there ways we could improve on existing projects to be more impactful? If there’s nothing already out there that addresses this challenge, ask yourself why not. What barriers exist that inhibit progress in this area? Does this challenge call for a technical solution? Maybe the issue is at heart an organizational or physical one, rather than something that a tech approach can solve. 
#### Identify existing alternatives

Take time to discuss as a group how people solve this problem today. Document all of the existing ways people solve the problem you want to solve.

![leancanvas-box1](https://cloud.githubusercontent.com/assets/100216/12663467/b887dd18-c5e5-11e5-9ac9-06aa2736118f.png)
# 

In short, no one is doing the same thing on the same scale or vision. 
In detail: 
1. I found some universities have community outreach programs to teach kids at local schools but nothing to bring music to healthcare facilities. 
2. One healthcare facility in Ohio let music students live for free in exchange of music performances for the residents
3. Music and Memory provide ipods for residents at healthcare facilities- they provide music, but not live music
4. A non-profit in the east coast bring in musicians to local healthcare facilities to perform and they do about 50 performances in a year, with 150 musicians volunteering over 10 years. For us, we will engage locally and then advance to nationally quickly by leveraging technology.
